moped-database/metadata/tables.yaml

```diff
@@ -3546,41 +3596,58 @@
           columns: '*'
         update:
           columns:
-            - capitally_funded
-            - is_deleted
-            - end_date
-            - added_by
-            - milestone_id
-            - project_id
-            - project_importance
-            - project_length
-            - project_order
             - project_sponsor
-            - timeline_id
             - task_order
+            - parent_project_id
```

Comment:
this is most all the columns on moped_project, except some of the ones we don't use (project_length) but most importantly it also omits "updated_at". Since we are updating updated_at with the REST connector, we don't need to trigger the REST connector again.